src/types/sourceTargetTypes.ts

```diff
@@ -26,7 +26,7 @@ export interface ApiPromiseConnectionType {
 
 export interface ConnectionChainInformation {
   chainNumber: string;
-  hasher?: (data: Uint8Array) => Uint8Array;
+  hasher: ((data: Uint8Array) => Uint8Array) | undefined;
```

Comment:
Is this required? It's almost exactly the same, but just less convenient (you always require the `hasher` field, but in case it's not needed it must have `undefined` value).